src/vcpkg/base/downloads.cpp

```diff
@@ -974,12 +980,15 @@ namespace vcpkg
                     fs, urls, headers, download_path, sha512, m_config.m_secrets, errors, progress_sink);
                 if (auto url = maybe_url.get())
                 {
+                    m_config.m_read_url_template.has_value() ? msg::println(msgAssetCacheMiss, msg::url = urls[0])
+                                                             : msg::println(msgDownloadingUrl, msg::url = urls[0]);
```

Comment:
```suggestion
                    msg::println(m_config.m_read_url_template.has_value() ? msgAssetCacheMiss : msgDownloadingUrl, msg::url = urls[0]);
```

Reply:
1. I didn't know I could do that lol
2. Can you explain why that is better?

Not saying I disagree, just want to know the reasoning :)

Reply:
I tried it but it didn't work